Extractor/ServerSubscriptionManager.cs

```diff
@@ -0,0 +1,105 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+using Cognite.OpcUa.History;
+using Microsoft.Extensions.Logging;
+
+using Opc.Ua;
+using Opc.Ua.Client;
+
+namespace Cognite.OpcUa
+{
+    public class ServerSubscriptionManager
```

Comment:
The name isn't great.

Reply:
Will work on coming up with a more descriptive name.